samples/apps/copilot-chat-app/webapp/src/components/chat/ChatWindow.tsx

```diff
@@ -111,6 +130,11 @@ export const ChatWindow: React.FC = () => {
         setIsEditing(!isEditing);
     };
 
+    const [selectedValue, setSelectedValue] = React.useState<TabValue>('chat');
```

Comment:
nit: change spelling to align with object being selected
```suggestion
    const [selectedTab, setSelectedTab] = React.useState<TabValue>('chat');
```